packages/cursorless-engine/src/actions/EditNew/EditNew.ts

```diff
@@ -1,8 +1,8 @@
+import { RangeUpdater } from "../..";
```

Comment:
Why the inconsistency in the import path?

Reply:
Getting auto-imported this way.  A bit annoying but I don't think it's worth the fight to fix it every time 🤷‍♂️

Reply:
Huh. Odd it only happened once though.
```suggestion
import { RangeUpdater } from "../../core/updateSelections/RangeUpdater";
```